### DISCUSSION TITLE
Solution to archive SOC logs to backup disk?

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#5446] [Unsupported Versions] [answered]
Hi all,

Do you have any suggestion to  to archive SOC logs to backup disk?

**Comment by Git-Me-Some-Hub** [comment, marked as answer]
I would recommend that you look at this discussion about backup the ES data with Snapshots. https://github.com/Security-Onion-Solutions/securityonion/discussions/4404 

There is also backups that happen on the /NSM disk 
https://docs.securityonion.net/en/2.3/backups.html?highlight=backups
